### PR TITLE
fix: ProducesResponseType

### PR DESCRIPTION
**Pull request by hhko**
`public async Task<Results<Ok,` method should have `[ProducesResponseType(StatusCodes.Status200OK)]`

### CODE REVIEW

**Comment by dr-marek-jaskula** [comment]
If we remove, for instance, a generic parameter 'AccessTokenResponse' from the 'ProducesResponseType' attribute, we would lose some swagger documentation:

With generic parameter:
with


Without generic parameter:
without

Its good to add them where they are missing (with generic parameter describing the response body), but I would not remove them.

**Comment by hhko** [comment]
@dr-marek-jaskula 
I also aggree with your opinions.
I only add `[ProducesResponseType(StatusCodes.Status200OK)]` where they are missing.

File: src/Shopway.Presentation/Controllers/UsersController.cs [hunk 1]
```diff
@@ -146,7 +146,7 @@ CancellationToken cancellationToken
 
     [HttpPost("[action]")]
     [Authorize]
-    [ProducesResponseType<Ok>(StatusCodes.Status200OK)]
```
Comment (dr-marek-jaskula) [inline src/Shopway.Presentation/Controllers/UsersController.cs:149]: Do not remove this generic parameter

Comment (hhko) [reply src/Shopway.Presentation/Controllers/UsersController.cs]: @dr-marek-jaskula 
I have applied your comment.
Thank you for your consideration.

**Review by dr-marek-jaskula — APPROVED** [review]

**Pull request merged**